docs/infrastructure_and_maintenance/clustering/ibexa_cloud_and_ddev.md

```diff
@@ -0,0 +1,94 @@
+---
+description: When you want to run locally an Ibexa Cloud project using DDEV.
+---
+
+# Ibexa Cloud and DDEV
+
+Two ways are available to run locally an Ibexa Cloud project using DDEV:
+
+- [Using the `ddev-platformsh` add-on](#simulate-ibexa-cloud-using-the-ddev-platformsh-add-on).
+- [Like other existing project, without this add-on](#simulate-ibexa-cloud-without-the-platformsh-add-on).
+
+## Simulate Ibexa Cloud using the `ddev-platformsh` add-on
+
+To configure the [`ddev/ddev-platformsh` add-on](https://github.com/ddev/ddev-platformsh), you'll need a [Platform.sh API Token](https://docs.platform.sh/administration/cli/api-tokens.html).
+
+`COMPOSER_AUTH` from PLatform.sh can't be properly used as JSON commas will be badly interpreted by `--web-environment-add` which would see them as multiple variables' separators. But, it must exist for Platform.sh hooks' scripts to work. An auth.json file can be used, see [Using an auth.json](../../getting_started/install_using_ddev.md#using-an-authjson) for more.
```

Comment:
```suggestion
`COMPOSER_AUTH` from Platform.sh can't be properly used, because JSON commas are incorrectly interpreted by `--web-environment-add`, which sees them as multiple variables' separators.
But the variable must exist for Platform.sh hooks' scripts to work. An `auth.json` file can be used, see [Using an auth.json](install_using_ddev.md#using-an-authjson) for more.
```